presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -63,10 +63,13 @@ class ConfigBase {
     return config_->values();
   }
 
+  void testingSetConfigOverrides(
+      const std::unordered_map<std::string, std::string>& configOverrides);
+
```

Comment:
Apart from E2E testing, it might be good for us to have some low level testing so we can have better control. Having testing prefix is to tell if this method is for testing purpose which is pretty common.